### ISSUE TITLE
Fix linter warnings for accessibility

### ISSUE THREAD

**Issue by Elblinator** [OpenSlides/openslides-client#3079]
When executing the "npx ng lint --fix" command several accessibility related linting warnings are reported. We should fix them. 
If they are fixed the behaviour of the client should change (at least slightly) to be more accessible.

What was changed or what different behaviour is to be expected should be documented here or in the PR which fixes the warnings
